src/lm/arpa-file-parser.h

```diff
@@ -100,24 +131,28 @@ class ArpaFileParser {
   /// Pure override that must be implemented to process current n-gram. The
   /// n-grams are sent in the file order, which guarantees that all
   /// (k-1)-grams are processed before the first k-gram is.
-  virtual void ConsumeNGram(const NGram&) = 0;
```

Comment:
It's [not quite precise](https://google.github.io/styleguide/cppguide.html#Type_Names) on that:

> Type names start with a capital letter and have a capital letter for each new word, with no underscores.

So, strictly speaking, this depends on whether you treat "n-gram" as one word or two. :)